src/libraries/Directory.Build.targets

```diff
@@ -285,4 +285,15 @@
     </When>
   </Choose>
 
+  <!-- The upfront vertical builds already generates the dependency file. -->
```

Comment:
This is true, but project changes that work in VS and don't require a vertical build (EG: adding a reference) would not be reflected in the deps file if we disable it.  Disabling in this way will make the build seem flaky.  Can you elaborate on what errors you were seeing?

Reply:
I agree with Eric. This would be a step backwards as we try to enable VS builds without an initial root build. Also disabling deps generation could easily influence test execution as xunit loads the test's deps file. I would very cautious about that.

Reply:
i debugged more and found the problem, its basically we are building all the tfms in vs but some of the project references are not build for some tfms like *-unix tfms.
i did a multiple scans over all the sln to check we didnt miss anything. There is not a good way to test this as we need a clean build for every project.
We need to add a ci protection here and a better way to catch these errors.